### PR TITLE
Add rock 3b image download

### PR DESCRIPTION
**Pull request by Maycides**
###### Review guide

To review this PR locally, please run the following commands:

```bash
PR= # the current PR number
git fetch origin pull/${PR}/head
git switch --detach FETCH_HEAD
yarn start
yarn start --locale en
# If you want to start 2 locales side-by-side, use the following command on Linux:
# https://github.com/facebook/docusaurus/issues/7377#issuecomment-1167192715
yarn start & DOCUSAURUS_GENERATED_FILES_DIR_NAME=.docusaurus/en yarn start --locale en --port 3001
kill %1
```

### CODE REVIEW

File: docs/rock3/images.md [hunk 1]
```diff
@@ -50,7 +50,15 @@ ROCK 3A 当前使用的是安卓 11，你可以从以下链接中下载镜像：
 </TabItem>
 <TabItem value="ROCK 3B">
 
-Comming soon.
+## ROCK 3B
+
+### 官方镜像
+
```
Comment (peterwang2050) [inline docs/rock3/images.md:56]: 不需要写首个

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/images.md [hunk 2]
```diff
@@ -49,7 +49,15 @@ Additional information:
 </TabItem>
 <TabItem value="ROCK 3B">
 
-Coming soon.
+## ROCK 3B
+
```
Comment (peterwang2050) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/images.md:53]: Renesas ? ? 

For more images ,你放到official下面, 客户还是会当作正式版本是用，比如 Ubuntu Server

File: docs/rock3/images.md [hunk 3]
```diff
@@ -50,7 +50,15 @@ ROCK 3A 当前使用的是安卓 11，你可以从以下链接中下载镜像：
...
+以下系统已由瑞莎官方测试验证：
+
+ROCK 3B：首个发布镜像：[Build 18](https://github.com/radxa-build/rock-3b/releases/download/b18/rock-3b_debian_bullseye_xfce_b18.img.xz)
```
Comment (Maycides) [inline docs/rock3/images.md:59]:
```suggestion
ROCK 3B：[Build 18](https://github.com/radxa-build/rock-3b/releases/download/b18/rock-3b_debian_bullseye_xfce_b18.img.xz)
```

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/images.md [hunk 4]
```diff
@@ -49,7 +49,15 @@ Additional information:
...
+### Official image
+
+The following systems have been officially tested and verified by Renesas:
```
Comment (Maycides) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/images.md:56]:
```suggestion
The following systems have been officially tested and verified by Radxa:
```

File: i18n/en/docusaurus-plugin-content-docs/current/rock3/images.md [hunk 5]
```diff
@@ -49,7 +49,15 @@ Additional information:
...
+
+ROCK 3B: The first release version: [Build 18](https://github.com/radxa-build/rock-3b/releases/download/b18/rock-3b_debian_bullseye_xfce_b18.img.xz)
```
Comment (Maycides) [inline i18n/en/docusaurus-plugin-content-docs/current/rock3/images.md:58]:
```suggestion
ROCK 3B:  [Build 18](https://github.com/radxa-build/rock-3b/releases/download/b18/rock-3b_debian_bullseye_xfce_b18.img.xz)
```

**Review by peterwang2050 — APPROVED** [review]

**Pull request merged**